markdown/chapters/meandeck/deckconstruction.md

```diff
@@ -0,0 +1,603 @@
+a copy of {{Watery Grave}} as a pseudo fifth copy. In this instance, if no basic lands
+are being run, the fetchland configuration can be changed to run any combination of
+{U}/x or {B}/x fetchlands. (Diversification is important to avoid {{Pithing Needle}}
+type effects. Conversely, in a more control-based list this can drop down as far as
+only one or two copies of Underground Sea with an Island dominant manabase being 
+favoured with copies of {{Urborg, Tomb of Yawgmoth}} being run as a mana fixer of choice.
+
+If splash colours are being run then it is generally accepted to run one {U}/x dual land
+of the splash colour and one {B}/x dual land of the splash colour.
+
+For example: In a BUG {B}{U}{G} based Doomsday list you might elect to run three
+Underground Seas, one {{Tropical Island}} and one {{Bayou}} as your dual lands of choice.
+
+<row variant="pile">{{!Underground Sea}} {{!Tropical Island}} {{!Bayou}}</row>
+
+The final land type to discuss (for this section) is your basic land count.
+Traditionally only two to three basics were run normally in a 2:1 ratio of
+Island to Swamp. More recently though some are opting to run only one of each,
+some only the Island and others (as mentioned above) are opting to run no
+basic lands at all. If you want to opt for a more control style build then
+you will probably need to look to increase your basic land count to four or 
+five. Normally you wouldn't consider running a splash basic such as a 
+mountain or forest however in the heavier control builds you might consider
+it.
+
+For example: In an Esper {W}{U}{B} based Doomsday list you might elect to run two
+Islands, two Swamps and one Plains as your basic lands of choice.
+
+<row variant="pile">{{!Island|RAV}} {{!Swamp|RAV}} {{!Plains|RAV}}</row>
+
+### 2.1 Deckbuilding Ratios - Lands
+
+The majority of Doomsday lists like to run between fourteen and seventeen lands
+as part of their core manabase. The number tends to decrease with a more aggressive
+playstyle and an increased number of free cycle effects and {{Lotus Petal}}s. It is
+normally split as eight fetch lands, two basics and five dual lands with numbers
+changing based on your preference and playstyle. You can supplement your land
+count with other toolbox lands like {{Cavern of Souls}} or {{Shelldock Isle}}
+however these will be discussed later as part of the Pile Cards section and should
+not be considered as part of your mana producing land count. Some people also
+opt to use other toolbox lands like {{Wasteland}} in their lists in an effort
+to be more disruptive. Again this should not factor as part of your mana
+requirements and instead be considered as Interaction.
+
+Let's consider some example mana bases for different styles of play for the deck.
+We shall present an aggressive, straight {U}{B} *Turbo* manabase, a more balanced
+Grixis {U}{B}{R} manabase and a control style Esper {W}{U}{B} manabase.
+
+#### {U}{B} *Turbo*
+
+- 4 Polluted Delta
```

Comment:
2 of each fetchland is better diversification

Reply:
Done